### PR TITLE
Added Webconf.asia

### PR DESCRIPTION
**Pull request by charisrooda**
Thanks a lot! :)

### CODE REVIEW

File: conferences/2018/general.json [hunk 1]
```diff
@@ -112,5 +112,16 @@
...
+  {
+    "name": "Webconf.asia",
+    "url": "https://webconf.asia",
+    "startDate": "2018-06-06",
+    "endDate": "2018-06-09",
+    "city": "Hong Kong",
+    "country": "Hong Kong",
+    "cfpUrl": "",
+    "cfpEndDate": "",
+    "twitter": "webconf_asia",
```
Comment (nimzco) [inline conferences/2018/general.json]: Thanks!
Can you remove this extra coma?

**Comment by nimzco** [comment]
Thanks! 👍

**Comment by nimzco** [comment]
No worries, I've added it by hand 👍

File: conferences/2018/general.json (hunk 1 above)
Comment (charisrooda) [reply conferences/2018/general.json]: Sorry, my bad. Thanks for fixing! :)

**Pull request closed without merging**